.env

```
@@ -0,0 +1,156 @@
+# .env (ADD NEW FILE)
+# Environment variables for Gemini Engine Components 5, 6, 7 Integration
+
+# ============================================================================
+# GEMINI API CONFIGURATION
+# ============================================================================
+GEMINI_API_KEY="AIzaSyD2WlV2eqqGMvrciq3qh5-F823L6kJlDyM"
+GEMINI_MODEL_NAME=gemini-1.5-pro
+GEMINI_BASE_URL=https://generativelanguage.googleapis.com/v1beta
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GEMINI_BASE_URL key should go before the GEMINI_MODEL_NAME key

(UnorderedKey)